src/commands/app/add/event.js

```diff
@@ -29,27 +31,63 @@ class AddEventCommand extends AddCommand {
     if (entries.length > 1) {
       this.error('Please use the \'-e\' flag to specify to which implementation you want to add events to.')
     }
+
     const configName = entries[0][0]
     const config = entries[0][1]
     const actionFolder = path.relative(config.root, config.actions.src)
     const configData = this.getRuntimeManifestConfigFile(configName)
 
-    const env = yeoman.createEnv()
-    // by default yeoman runs the install, we control installation from the app plugin
-    env.options = { skipInstall: true }
-    const eventsGen = env.instantiate(generators['add-events'], {
-      options: {
-        'skip-prompt': flags.yes,
-        'action-folder': actionFolder,
-        'config-path': configData.file,
-        'full-key-to-manifest': configData.key,
-        // force overwrites, no useless prompts, this is a feature exposed by yeoman itself
-        force: true
+    const templateOptions = {
+      'skip-prompt': false,
```

Comment:
why not flags.yes?

Reply:
for events templates, we always need the skip-prompt to be false, true will not make sense as there are no default values that can be added to the app.

Reply:
ok then maybe let's add a note to the flag that it will not have any effect when allow-events-template is set

Reply:
thanks, added. also added a related issue to be fixed in https://github.com/adobe/generator-app-common-lib/issues/38

Reply:
nice thanks